test/read_evidence_performance.py

```diff
@@ -0,0 +1,53 @@
+"""
+Simple script to measure performance of the read_evidence module.
+
+Usage:
+    %(prog)s /path/to/reads.bam <region1> <region2> ... --num-loci-per-region X
+
+Example:
+    %(prog)s my.bam 8:101723999-101725999 --num-loci-per-region 2000
+
+"""
+import argparse
+import sys
+import random
+import time
+
+import varcode
+import varcode.read_evidence
+from varcode.locus import Locus
+
+def parse_region(s):
+    (contig, rest) = s.split(":")
+    (start, end) = rest.split("-")
+    return varcode.Locus.from_inclusive_coordinates(
+        contig, int(start), int(end))
+
+def go(argv):
+    parser = argparse.ArgumentParser(usage=__doc__)
```

Comment:
Sure, I can put the parser at global scope.

By doing `sys.argv[1:]` myself I have a function that I could theoretically call from other code, like tests, without assuming anything about the global argv